src/model/Subscription.js

```diff
@@ -19,7 +19,13 @@ const creatableField = [
   "default_branch"
 ];
 
-const modifiableField = ["plan", "environments", "storage", "big_dev"];
+const modifiableField = [
+  "plan",
+  "environments",
+  "storage",
+  "big_dev",
+  "agency_site"
```

Comment:
If this field is only available on OrganizationSubscription, i think it could be better to add it only on this model https://github.com/platformsh/platformsh-client-js/blob/master/src/model/OrganizationSubscription.js

Reply:
This field is in accounts so available in Subscription as well